compose/metrics/client.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, command,
                  context_type=None, status=Status.SUCCESS,
                  source=MetricsSource.CLI, uri=None):
         super().__init__()
-        self.command = "compose " + command if command else "compose --help"
+        self.command = ("compose " + command).strip() if command else "compose --help"
```

Comment:
I suppose this is to leave `self.command = "compose"` value in case of NoSuchCommand exception (when command = "") ?

Reply:
Yes. In the case of `NoSuchCommand` the variable will have an empty string leaving a trailing space on the final string. The `strip()` is to remove it in this case.